### ISSUE TITLE
Undo button may be disabled during animations

### ISSUE THREAD

**Issue by Btimes2** [Sasszem/13#72]
Clicking on the Undo button during animation should be simply ignored, instead of waiting for the animation to finish and then quickly do an undo. I think ignoring this would provide a smoother game experience. If I click on it now, then the Undo is so fast that it is hard to follow what happens on the screen. And I think the Undo will be used rarely, so lots of undo clicks will be accidental, too.

**Comment by Sasszem** [comment]
<del>Clicking "undo" can also crash the game if unfortunate enough...</del>
Whoopsie, that HAD been fixed.
